AllCoreDevs-Meetings/Meeting 150.md

```diff
@@ -0,0 +1,576 @@
+# Ethereum Core Devs Meeting #150 Notes
+### Meeting Date/Time: 2022-Nov.-24- 14:00 UTC
+### Meeting Duration: 01:47:28
+### Audio/Video of the Meeting: [Recording](https://www.youtube.com/watch?v=HX_Zr_gVeOE)
+### Moderator: Tim Beiko
+
+
+
+# **Summary:**
+- We discussed the state of testnets and what to do with them going forward. Only specific decision is agreeing to fully shut down Ropsten by EOY. Broader testnet proposal to be discussed here: https://ethereum-magicians.org/t/proposal-predictable-ethereum-testnet-lifecycle/11575 
+- We went over the EIPs we've been discussing for a while now and discussed which should be moved to CFI... and what CFI means! No strong consensus on the latter, "definition" is here (https://github.com/ethereum/execution-specs/tree/master/network-upgrades#definitions). With 4844 and EOF (all 5 EIPs) being on devnets already, they've been moved to CFI. For 4758, people wanted to see a better elaboration of the ways to deal with contract breakage before moving it to CFI. 2537 was also moved to CFI... but it's been CFI'd since Berlin. 
```

Comment:
```suggestion
- Client teams went over the EIPs under discussion for a while now. No strong consensus on "what CFI means", "definition" is here (https://github.com/ethereum/execution-specs/tree/master/network-upgrades#definitions). With 4844 and EOF (all 5 EIPs) being on devnets already, they've been moved to CFI. For 4758, people wanted to see a better elaboration of the ways to deal with contract breakage before moving it to CFI. 2537 was also moved to CFI... but it's been CFI'd since Berlin. 
```